Products/ZenRRD/zentestcommand.py

```diff
@@ -16,29 +16,24 @@
 
 __version__ = "$Revision$"[11:-2]
 
-import os
-import popen2
-import fcntl
-import time
-import sys
-import select
-import logging
-import signal
 from copy import copy
-log = logging.getLogger("zen.zentestcommand")
+import logging
+import sys
 
-import Globals
-from Products.ZenUtils.ZenScriptBase import ZenScriptBase
-from Products.DataCollector.SshClient import SshClient
+import Globals # noqa
+from Products.ZenUtils import Utils
+from Products.ZenUtils import ZenScriptBase
```

Comment:
hm, I see both cases in Zenoss code and PEP8 allows both cases too.
Personally, I prefer to use `import module`, not `from module import func` but I understand that this question may be a part of holywar and the difference is mainly subjective so I'll agree with any your suggestion.